lib/assets.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
 
-var merge = require('lodash').merge;
-var gulpif = require('gulp-if');
-var imagemin = require('gulp-imagemin');
+import {merge} from 'lodash';
+import gulpif from 'gulp-if';
+import imagemin from 'gulp-imagemin';
 
-var defaults = {
+const defaults = {
```

Comment:
I think @erikjung already covered this, but this doesn't feel like a good place for the `const` keyword, as the object here gets mutated later.